templates/search/clustered_graph.html

```diff
@@ -0,0 +1,347 @@
+{% load static %}
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Force-Directed Layout</title>
```

Comment:
This file I won't review as I understand this is a very prototype thing which will not be deployed for Freesound users.